content/2015.05.cliquet-permissions-en.rst

```diff
@@ -0,0 +1,535 @@
+|                |                        | content of all objects in the     |
+|                |                        | *bucket*.                         |
+|                +------------------------+-----------------------------------+
+|                | `collections:create`   | Permission to create              |
+|                |                        | collections in the *bucket*.      |
+|                +------------------------+-----------------------------------+
+|                | `groups:create`        | Permission to create groups       |
+|                |                        | in the *bucket*.                  |
++----------------+------------------------+-----------------------------------+
+| ``collection`` | `write`                | Permission to administrate all    |
+|                |                        | objects in the collection.        |
+|                +------------------------+-----------------------------------+
+|                | `read`                 | Permission to consult all         |
+|                |                        | objects in the collection.        |
+|                +------------------------+-----------------------------------+
+|                | `records:create`       | Permission to create new          |
+|                |                        | records in the collection.        |
++----------------+------------------------+-----------------------------------+
+| ``record``     | `write`                | Permission to modify or share     |
+|                |                        | the record.                       |
+|                +------------------------+-----------------------------------+
+|                | `read`                 | Permission to read the record.    |
+|                |                        |                                   |
++----------------+------------------------+-----------------------------------+
+| ``group``      | `write`                | Permission to administrate the    |
+|                |                        | group.                            |
+|                +------------------------+-----------------------------------+
+|                | `read`                 | Permission to know the members of |
+|                |                        | the group.                        |
++----------------+------------------------+-----------------------------------+
+
+
+*principals*
+------------
+
+Actors connecting to the storage service can authenticate themselves.
+
+They then receive a list of *principals*:
+
+- ``Everyone``: the *principal* given to all actors (authenticated or not);
+- ``Authenticated``: the *principal* given to all authenticated actors;
+- A *principal* identifying the actor, for instance ``fxa:32aa95a474c984d41d395e2d0b614aa2``
+
+In order to avoid identifiers collisions, the actor *principal* is built from
+the authentication type she used (``system``, ``basic``, ``ipaddr``, ``hawk``,
```

Comment:
As we previously mentioned that a principal can be a computer and was a generic abstraction of actors, *the authentication type it used*